src/messages/MPGStats.h

```diff
@@ -61,6 +61,10 @@ class MPGStats : public PaxosServiceMessage {
     paxos_decode(p);
     decode(fsid, p);
     decode(osd_stat, p);
+    if (osd_stat.num_osds == 0) {
```

Comment:
May be set to 1 in osd_stat_t::decode when version is <12?

Reply:
i thought about that, but it is more confusing that way: we want the ctor to be zero so that these structures sum properly, and if we encode/decode the sum anywhere it will end up wrong.  better to explicitly infer it when the osd_stat_t is collected from the OSD.